package.json

```diff
@@ -144,9 +146,12 @@
   },
   "browserslist": {
     "production": [
-      ">0.2%",
+      ">0.3%",
```

Comment:
See https://github.com/Uniswap/interface/issues/2367. This should be resolved before committing a change to browserslist. Would you mind commenting with your rationale in that issue?